### ISSUE TITLE
Remove or mark "outdated" entries with no recent release

### ISSUE THREAD

**Issue by theperiscope** [awesome-selfhosted/awesome-selfhosted#2922]
TL;DR

I'd like to not see (or see as clearly marked) entries which have had no release (i.e. outdated) after some pre-determined period of time.

Details

I clicked through the "Pastebins" section in an effort to find a self-hosted pastebin alternative. Lots of the options have not had a release in 12+ months. (e.g. 0bin, privatebin, dpaste, exbin, etc.) with few exceeding 24 months. I found similar issues in other categories as well. Overall, it is difficult for me to self-host and trust something that's possibly not being updated and I think it could benefit the repository if entries are evaluated on a periodic basis, or alternatively each entry in the list was prefixed with a "last release" date (perhaps included via some sort of automated script).
